cobigen/cobigen-propertyplugin/src/main/java/com/devonfw/cobigen/propertyplugin/PropertyMerger.java

```diff
@@ -53,27 +60,25 @@ public String getType() {
 
     @Override
     public String merge(File base, String patch, String targetCharset) throws MergeException {
-
         Properties baseProperties = new Properties();
-        try (FileInputStream in = new FileInputStream(base);
-            InputStreamReader reader = new InputStreamReader(in, targetCharset)) {
-            baseProperties.load(reader);
+        try {
+            baseProperties.load(new InputStreamReader(new FileInputStream(base), targetCharset));
         } catch (IOException e) {
             throw new MergeException(base, "Could not read base file.", e);
         }
-
         Properties patchProperties = new Properties();
-        try (ByteArrayInputStream inStream = new ByteArrayInputStream(patch.getBytes())) {
-            patchProperties.load(inStream);
+        try {
+            patchProperties.load(new ByteArrayInputStream(patch.getBytes()));
         } catch (IOException e) {
             throw new MergeException(base, "Could not read generated patch.", e);
         }
-
         Set<Object> conflicts = getConflictingProperties(baseProperties, patchProperties);
-        try (FileInputStream in = new FileInputStream(base);
-            InputStreamReader reader = new InputStreamReader(in, targetCharset);
-            BufferedReader br = new BufferedReader(reader)) {
-            return concatContents(conflicts, br, patch);
+        try (FileInputStream stream = new FileInputStream(base);
+            BufferedInputStream bis = new BufferedInputStream(stream); 
+        	InputStreamReader reader = new InputStreamReader(bis, targetCharset)){
+            BufferedReader br = new BufferedReader(reader);
+            String lineDelimiter  = SystemUtil.determineLineDelimiter(bis, reader);
```

Comment:
This has to be changed to determineLineDelimiter(Path) due to changes made to the method in the core
Check #1252